src/engines/kotor/types.h

```diff
@@ -152,6 +152,19 @@ enum SubRace {
 	kSubRaceWookie
 };
 
+enum EquipmentSlot {
+	kEquipmentSlotNone    = 0,
+	kEquipmentSlotHead    = 1,
+	kEquipmentSlotBody    = 2,
+	kEquipmentSlotImplant = 4,
+	kEquipmentSlotHands   = 8,
+	kEquipmentSlotWeaponL = 16,
+	kEquipmentSlotWeaponR = 32,
+	kEquipmentSlotArmL    = 128,
+	kEquipmentSlotArmR    = 256,
+	kEquipmentSlotBelt    = 1024
```

Comment:
Can you rather use
```
kEquipmentSlotNone    = 0,
kEquipmentSlotHead    = 1U << 0,
kEquipmentSlotBody    = 1U << 1,
kEquipmentSlotImplant = 1U << 2,
...
```

for that, like here https://github.com/xoreos/xoreos/blob/master/src/engines/kotor/types.h#L34 ?